pkg/services/export/export_sys_playlists.go

```diff
@@ -37,7 +37,7 @@ func exportSystemPlaylists(helper *commitHelper, job *gitExportJob) error {
 		}
 
 		gitcmd.body = append(gitcmd.body, commitBody{
-			fpath: filepath.Join(helper.orgDir, "system", "playlists", fmt.Sprintf("%s-playlist.json", playlist.UID)),
+			fpath: filepath.Join(helper.orgDir, "system", "playlists", fmt.Sprintf("%s-playlist.json", playlist.Uid)),
```

Comment:
@sdboyer -- thoughts on if "UID" should be all caps in the export?

Reply:
it wouldn't be terribly difficult to write an AST transform for it. Not sure how much true value we really get, though?

Reply:
i should rephrase - i can definitely see value. But not enough that i would prioritize it above the other things on my plate. If somebody else wants to do it, i'd be happy to guide & review :)